File: Service/Implement/DanhMucNguyenVatLieuService.cs
namespace Service.Implement
{
    public class DanhMucNguyenVatLieuService : BaseService<DanhMucNguyenVatLieu, IDanhMucNguyenVatLieuRepository>
    , IDanhMucNguyenVatLieuService
    {
    private readonly IDanhMucNguyenVatLieuRepository _DanhMucNguyenVatLieuRepository;
    public DanhMucNguyenVatLieuService(IDanhMucNguyenVatLieuRepository DanhMucNguyenVatLieuRepository) : base(DanhMucNguyenVatLieuRepository)
    {
    _DanhMucNguyenVatLieuRepository = DanhMucNguyenVatLieuRepository;
    }
    }
    }

File: Map/src/app/shared/DanhMucNguyenVatLieu.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { DanhMucNguyenVatLieu } from 'src/app/shared/DanhMucNguyenVatLieu.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class DanhMucNguyenVatLieuService extends BaseService{
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "DanhMucNguyenVatLieu";
    }
}

